h2o-r/h2o-package/R/communication.R

```diff
@@ -800,7 +800,12 @@ h2o.show_progress <- function() assign("PROGRESS_BAR", TRUE, .pkg.env)
       }
 
       job = jobs[[1]]
-
+      if(verboseModelScoringHistory){
```

Comment:
This effectively modifies the the polling interval, right?

Can we avoid the sleep by moving the logic after the existing Sys.sleep (pollingInterval)?